app/assets/stylesheets/variables.scss

```diff
@@ -66,3 +66,7 @@ $secondary:          $admin-steel-blue;
 $content-background: $very-light-grey;
 $content-shadow:     $light-grey;
 $content-line:       $grey;
+
+// Units
+$border-radius: 4px;
```

Comment:
I noticed that you added this `$border-radius` variable but it’s not being called anywhere in the app — was this intentional? Or did you want to use it someplace but haven’t added it yet? If so I’d suggest removing it from this PR and adding it later!

Reply:
Good point, removed this. :)